### ISSUE TITLE
Doesn't work with Jupyter Lab version 3.0.1

### ISSUE THREAD

**Issue by jithin-k-sreedharan** [quigleyj97/jupyterlab-spreadsheet#30]
Running the installation command `jupyter labextension install jupyterlab-spreadsheet` throws the following error:
```
An error occured.
ValueError: The extension "jupyterlab-spreadsheet" does not yet support the current version of JupyterLab.
```

**Comment by NQuigley02** [comment]
Working on this now

**Comment by quigleyj97** [comment]
Released `jupyterlab-spreadsheet@0.4.0`. You should now be able to use this extension in JupyterLab 3 after upgrading to this version.
